scheduler/src/cook/mesos/api.clj

```diff
@@ -1108,7 +1116,7 @@
                        (catch Exception e
                          (log/warn e "Malformed raw api request")
                          [true {::error (.getMessage e)}]))))
-
+     :allowed? (partial job-create-allowed? is-authorized-fn)
```

Comment:
Technically, it's possible, but I think it's more effort than it's worth. I think it would require starting up cook configured with a different `:authorization-fn` that would specifically disallow creation based on something unique to the test.